src/lib/public/copy/copy.ts

```diff
@@ -0,0 +1,13 @@
+import { toast, ToastType } from '$lib/components/toaster/toasts';
+
+export default function copy(
```

Comment:
Nit: To remain consistent with the rest of the repository, we could export without the `default` keyword.
```suggestion
export function copy(
```

Reply:
done